### PR TITLE
Add default feature ms5611, add alternative feature for ms5607

### PR DESCRIPTION
**Pull request by richardeoin**
ms5607 is almost identical, except the pressure calibration constants
need to be shifted one bit differently.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -88,11 +95,11 @@ where
 
         let dt = (raw_sample.temperature as i32)
             - ((self.coeffs.get_data(CoefficientsAddr::COEFF_5) as i32) << 8);
-        let mut off = ((self.coeffs.get_data(CoefficientsAddr::COEFF_2) as i64) << 16)
-            + (((self.coeffs.get_data(CoefficientsAddr::COEFF_4) as i64) * (dt as i64)) >> 7);
+        let mut off = ((self.coeffs.get_data(CoefficientsAddr::COEFF_2) as i64) << 17)
```
Comment (Arkaleks) [inline src/lib.rs]: Shouldn't this line also use the same pattern as `get_compensated_sample` ?
`
        let off = ((self.coeffs.get_data(CoefficientsAddr::COEFF_2) as i64) << offset_sh.0)
            + (((self.coeffs.get_data(CoefficientsAddr::COEFF_4) as i64) * (dt as i64))
                >> offset_sh.1);
`

File: src/lib.rs [hunk 2]
```diff
@@ -88,11 +95,11 @@ where
...
+            + (((self.coeffs.get_data(CoefficientsAddr::COEFF_4) as i64) * (dt as i64)) >> 6);
 
-        let mut sens = ((self.coeffs.get_data(CoefficientsAddr::COEFF_1) as i64) << 15)
-            + (((self.coeffs.get_data(CoefficientsAddr::COEFF_3) as i64) * (dt as i64)) >> 8);
+        let mut sens = ((self.coeffs.get_data(CoefficientsAddr::COEFF_1) as i64) << 16)
```
Comment (Arkaleks) [inline src/lib.rs]: Same comment as above ?

**Comment by Arkaleks** [comment]
Thanks for the contribution, please review my comment. I think you forget to select the correct offset in `get_second_order_sample`

**Review by Arkaleks — CHANGES_REQUESTED** [review]

File: src/lib.rs (hunk 1 above)
Comment (richardeoin) [reply src/lib.rs]: Thanks, good spot. I've actually broken this out into a new function called `get_temperature_compensation`. This function is now the only place where ms5607 / ms5611 differ.

**Comment by richardeoin** [comment]
I also added a test for the MS5607!

**Pull request merged**